integration/options.go

```diff
@@ -57,6 +57,25 @@ const (
 
 	// defaultUseTChannelClientForTruncation determines whether we use the tchannel client for truncation by default.
 	defaultUseTChannelClientForTruncation = true
+
+	// defaultRepairThrottle determines the throttle per namespace repair by default.
+	defaultRepairThrottle = time.Duration(1 * time.Minute)
+
+	// defaultRepairTimeJitter determines the repair jitter by default.
+	defaultRepairTimeJitter = time.Duration(1 * time.Second)
+
+	// defaultRepairInterval determines the interval between repairs by default.
+	defaultRepairInterval = time.Duration(2 * time.Minute)
+
+	// defaultNumShards sets the default number of shards.
+	defaultNumShards = 1024
+
+	// defaultFetchSeriesBlocksBatchSize sets the number of series blocks to fetch in batch, by default.
+	defaultFetchSeriesBlocksBatchSize = 0
+
+	// defaultFetchSeriesBlocksBatchConcurrency sets the number of series blocks to fetch in
+	// batch concurrently, by default.
+	defaultFetchSeriesBlocksBatchConcurrency = 0
```

Comment:
Good catch. If `0`, we fall back to the admin options default. They're used [here](https://github.com/m3db/m3db/pull/214/files/4bf93c374fdd55cb46960d13334e205abc6eb500#diff-34fc2068cceaa93f8d6b312c44397541R186)

I had the default value set to make it explicit, but it's probably a lot cleaner to just rely on 0 being the default value if the field is unset. Will address in the next patch.